### DISCUSSION TITLE
v3.13.0 (rc1)

### DISCUSSION THREAD

**Discussion by mneunomne** [dhowe/AdNauseam#2082] [General]
- Merged to uBlock 1.40.4
- Implemented [Strict Block List](https://github.com/dhowe/AdNauseam/wiki/Developer-FAQ#what-is-the-strict-block-list) https://github.com/dhowe/AdNauseam/issues/2036
- Implemented new three button state for 'active' 'strict' 'disabled', with popup selecting domain or page scope for the disable button.
- Adjusts in the Ad-Vault

<hr /><em>This discussion was created from the release <a href='https://github.com/dhowe/AdNauseam/releases/tag/'>v3.13.0 (rc1)</a>.</em>

**Comment by ice-fly** [comment]
**Chrome:** 100.0.4896.127 (Official Build) (64-bit) (cohort: Stable) **OS:** Windows 10 Version 21H2
**Adnauseam:** 3.13.0.1 _(loaded as Unpacked extension)_
Running reasonably well. No crashes or unexpected errors. 👍

**Reply by mneunomne** [reply]
@ice-fly  great, thanks! Any feedback on the strict-block function from the menu?

**Reply by ice-fly** [reply]
I've yet to find a niche where I need it but it will be something to add to the arsenal when troubleshooting lag on browser. Not much reason for my use case to toggle it but it's still a nice feature. UI implementation is not bad. Thanks again for Devving this!
